src/icons/Send.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
```

Comment:
can you reuse existing icons? I am pretty sure you just need to adjust colors and size and add a stroke

Reply:
I tried, it doesn't look right. There's something different between the path's that changes how the stroke appears

Reply:
surprising given that the figma references the base icon. Maybe the base icon itself is wrong. Anyway, can we group all the home screen specific ones into a folder? So we don't have 2 icons with the same name which could be confusing